README.md

```diff
@@ -63,12 +63,12 @@ function getUserPosts(id, blogId, limit, offset) {
 }
 ~~~
 
-As you can see, this minimal example is already rather hard to read. It is also incorrect:
+As we can see, this minimal example is already rather hard to read. It is also incorrect:
 
-- I forgot that there was a trailing slash at the end of the `API_URL` constant so the slash got duplicated (`https://api.example.com//users`)
+- One common issue is of trailing slashes, `API_URL` has a trailing slash in the example. This will lead to URL containing duplicated slashes (`https://api.example.com//users`).
 - The embedded values need to be escaped using `encodeURIComponent`
 
-I can use the built-in `URL` class to prevent duplicate slashes and `URLSearchParams` to escape the query string. But I still need to escape all path parameters manually.
+We can use the built-in `URL` class to prevent duplicate slashes and `URLSearchParams` to escape the query string. But we still need to escape all path parameters manually.
```

Comment:
As I said earlier, I would like to keep the first person voice. Could you please undo the changes related to it?